events_ajax.php

```diff
@@ -31,10 +31,10 @@
 require_once 'includes/validate.php';
 
 // Load Doc classes for attachments and comments
-require_once 'includes/classes/Doc.php';
 require_once 'includes/classes/DocList.php';
```

Comment:
I believe we had a bug report about this order.  DocList requires Doc before it, right?